test/api/assignments_test.rb

```diff
@@ -130,13 +130,13 @@ def test_notify_team_members_about_submission
   end
 
   def test_api_rejects_duplicates
-    Attempt.new(alice, Iteration.new({'THE CODE' => 'ruby/one/code.rb'}, 
+    Attempt.new(alice, Iteration.new({ "THE CODE" => "ruby/one/code.rb" }, 
     {track: 'ruby', slug: 'one' })).save
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
         solution = { "THE CODE" => "ruby/one/code.rb" }
         post '/user/assignments', { key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb', 
-          'language' => 'ruby', 'problem' => 'one', solution: solution }.to_json
+        'language' => 'ruby', 'problem' => 'one', solution: solution }.to_json
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.